fili-core/src/test/groovy/com/yahoo/bard/webservice/druid/serializers/DimensionToDefaultDimensionSpecSpec.groovy

```diff
@@ -0,0 +1,96 @@
+// Copyright 2016 Yahoo Inc.
+// Licensed under the terms of the Apache license. Please see LICENSE file distributed with this work for terms.
+package com.yahoo.bard.webservice.druid.serializers
+
+import static com.yahoo.bard.webservice.data.time.DefaultTimeGrain.HOUR
+import static org.joda.time.DateTimeZone.UTC
+
+import com.yahoo.bard.webservice.data.DruidQueryBuilder
+import com.yahoo.bard.webservice.data.PartialDataHandler
+import com.yahoo.bard.webservice.data.QueryBuildingTestingResources
+import com.yahoo.bard.webservice.data.dimension.Dimension
+import com.yahoo.bard.webservice.data.metric.LogicalMetric
+import com.yahoo.bard.webservice.data.metric.mappers.NoOpResultSetMapper
+import com.yahoo.bard.webservice.data.volatility.DefaultingVolatileIntervalsService
+import com.yahoo.bard.webservice.druid.model.query.DruidAggregationQuery
+import com.yahoo.bard.webservice.table.resolver.DefaultPhysicalTableResolver
+import com.yahoo.bard.webservice.web.DataApiRequest
+
+import com.fasterxml.jackson.databind.ObjectMapper
+
+import org.joda.time.DateTime
+import org.joda.time.Hours
+import org.joda.time.Interval
+
+import spock.lang.Specification
+
+/**
+ * Testing dimension serialization to default dimension spec.
+ */
+class DimensionToDefaultDimensionSpecSpec extends Specification {
+
+    static ObjectMapper objectMapper
+    static QueryBuildingTestingResources resources
+    static Set<Interval> intervals
+    static DruidQueryBuilder builder
+    static DefaultPhysicalTableResolver resolver
+
+    def initDefault(DataApiRequest apiRequest, Dimension dim) {
+        intervals = [new Interval(new DateTime("2015"), Hours.ONE)]
+        LogicalMetric lm1 = new LogicalMetric(resources.simpleTemplateQuery, new NoOpResultSetMapper(), "lm1", null)
+
+        apiRequest.getTable() >> resources.lt12
+        apiRequest.getGranularity() >> HOUR.buildZonedTimeGrain(UTC)
+        apiRequest.getTimeZone() >> UTC
+        apiRequest.getDimensions() >> ([dim] as Set)
+        apiRequest.getLogicalMetrics() >> ([lm1] as Set)
+        apiRequest.getIntervals() >> intervals
+        apiRequest.getFilterDimensions() >> []
+        apiRequest.getTopN() >> OptionalInt.empty()
+        apiRequest.getSorts() >> ([] as Set)
+        apiRequest.getCount() >> OptionalInt.empty()
+    }
+
+    def setupSpec() {
+        objectMapper = new ObjectMapper()
+        resources = new QueryBuildingTestingResources()
+        resolver = new DefaultPhysicalTableResolver(new PartialDataHandler(), new DefaultingVolatileIntervalsService())
+        builder = new DruidQueryBuilder(resources.logicalDictionary, resolver)
+    }
+
+    def "Testing serialization with equal apiName and physicalName, serialize to apiName string"() {
```

Comment:
In general, we try to name tests in a way that describe the behavior we're testing for as clearly and succinctly as possible, often in a "given, when, then" style that mimics the contents of the blocks in the test method.

Specifically for this test name, here are some improvements:
- Since spock tests are _all_ testing something, having "Testing" in the name doesn't add anything
- It would be good to indicate _what_ is supposed to have the same apiName and physicalName (Dimension, in this case)
- Can drop "string" from the end. Since an apiName _is_ a string, there's nothing really gained by having it in the name